### PR TITLE
allow pvc by default

### PR DESCRIPTION
**Pull request by pweil-**
Fixes: https://github.com/openshift/origin/issues/8297

@liggitt @smarterclayton - will work on an e2e test that uses pvcs

### CODE REVIEW

**Comment by smarterclayton** [comment]
@pweil- storage team really needs to own that - can you sync with @bchilds
and @markturansky?

**Comment by liggitt** [comment]
unit test to ensure all sccs include the fstypes we've determined should always be allowed by default, then LGTM

**Comment by pweil-** [comment]
test added to check bootstrapped constraints for `[]kapi.FSType{kapi.FSTypeEmptyDir, kapi.FSTypeSecret, kapi.FSTypeDownwardAPI, kapi.FSTypeConfigMap, kapi.FSTypePersistentVolumeClaim}` or `FSTypeAll`

File: pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go [hunk 1]
```diff
@@ -35,6 +38,12 @@ func TestBootstrappedConstraints(t *testing.T) {
 		if !reflect.DeepEqual(u, constraint.Users) {
 			t.Errorf("unexpected user access for %s.  Found %v, wanted %v", constraint.Name, constraint.Users, u)
 		}
+
+		for _, expectedVolume := range expectedVolumes {
```
Comment (liggitt) [inline pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go]: or has "*", right?

Comment (liggitt) [inline pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go]: nm, missed the check below

**Comment by liggitt** [comment]
LGTM

**Comment by pweil-** [comment]
[merge]

**Comment by smarterclayton** [comment]
[test]

**Pull request merged**